Apps/Common/src/AspNetConfiguration/Modules/OdrConfiguration.cs

```diff
@@ -74,7 +74,7 @@ private static Uri GetBaseUrl(OdrConfig odrConfig)
                 ? "/pgw/patientGateway"
                 : "/odr";
 
-            return new(new(odrEndpoint), pathPrefix);
+            return new(odrEndpoint.TrimEnd('/') + pathPrefix);
```

Comment:
NIT: or use UriBuilder?

Reply:
It doesn't seem to have any methods to help combine paths.